js/privatebin.js

```diff
@@ -43,10 +43,21 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         var me = {};
 
         /**
+         * list of UserAgents (parts) known to belong to a bot
+         *
+         * @private
+         * @enum   {Object}
+         * @readonly
+         */
+        var BadBotUA = [
+            'Bot',
+            'bot'
+        ];
+
+        /**
          * character to HTML entity lookup table
          *
          * @see    {@link https://github.com/janl/mustache.js/blob/master/mustache.js#L60}
-         * @name Helper.entityMap
```

Comment:
But these are private, internal variables so they actually do not have any name. You cannot use them from the "outside". The only valid name would probably be "entityMap", but we do not have to document it via "@name"…